worker/codalabworker/run_manager.py

```diff
@@ -120,19 +120,22 @@ def memory_bytes(self):
 
 
 class Reader(object):
+
+    read_handlers = {
```

Comment:
`self` is not defined in this context as handler is a class variable. As a result this doesn't work in its current form. I guess one way to make it work would be to build this handle dict in `__init__` and make it an instance variable.

Reply:
good idea

Reply:
done